query/executor.go

```diff
@@ -289,6 +291,42 @@ func (e *Executor) WithLogger(log *zap.Logger) {
 	e.TaskManager.Logger = e.Logger
 }
 
+func (e *Executor) WithLogWriter(log *zap.Logger, path string, format string) error {
```

Comment:
I'd rather just have this take a `*zap.Logger` and rely on the caller to have it setup properly. This gives us a lot more flexibility.

I'm also thinking `WithQueryLogger` is a more descriptive name. This will only log queries, but `WithLogWriter` implies all logging will go into `log`.

Reply:
Actually - I could probably remove this and then just update the config object below. Then in `plutonium` if the config for `query-log-path` is set I can alter the zap logger while bootstrapping the influxd server and pass it in to `WithLogger`.